.kitchen.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+platforms:
+  - name: local
+
+provisioner:
+  name: terraform
+
+verifier:
+  name: terraform
+  color: false
+  fail_fast: false
+
+suites:
+  - name: forseti_integration_testing
+    driver:
+      name: terraform
+      root_module_directory: integration/fixtures/forseti_integration_testing
```

Comment:
Seems like we don't need to have a specific `forseti_integration_testing` reference here, as there is only one fixture here.

Reply:
So, the root module should be `integration_tests/fixtures`.

Reply:
Consider it done

Reply:
Done 9 days ago